src/App.tsx

```diff
@@ -1,20 +1,26 @@
 import React, { Component } from "react";
 import { getCompliments, getAdvice } from "./ApiCalls";
-import "./App.css";
-import { Button } from "./components/Button/Button";
-import { AppContainer } from "./components/AppContainer/AppContainer";
+import { Navigation } from "./components/Navigation/Navigation";
+import { Routes } from "./components/Routes/Routes";
 
-interface Props {}
+
+
+
+
+interface Props { }
 interface State {
   compliment: string;
   advice: string;
 }
 
 class App extends Component<Props, State> {
-  state: State = {
-    compliment: "",
-    advice: "",
-  };
+  constructor(props: Props) {
```

Comment:
Look at us refactoring to the correct form of State for TS with a constructor and props!